framework/include/mfem/equation_systems/equation_system.h

```diff
@@ -0,0 +1,186 @@
+#ifdef MFEM_ENABLED
+
+#pragma once
+#include "libmesh/ignore_warnings.h"
+#include "mfem/miniapps/common/pfem_extras.hpp"
+#include "libmesh/restore_warnings.h"
+#include "boundary_conditions.h"
+#include "MFEMContainers.h"
+#include "MFEMKernel.h"
+#include "ScaleIntegrator.h"
+
+namespace platypus
```

Comment:
- resolve namespace
- camel case methods
- move public data that doesn't need to be public